components/sysutils/virtualbox/virtualbox-additions.p5m

```diff
@@ -16,7 +16,7 @@
 #
 
 set name=pkg.fmri value=pkg:/system/virtualbox/virtualbox-additions@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
-set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=pkg.summary value="VirtualBox Guest Additions - provide host/guest integration (install only in VirtualBox guest)"
```

Comment:
Please, remove (install only) comment - we can't avoid all user errors by moving documentation to package summaries.